test/cut/test_cut_mixing.py

```diff
@@ -324,6 +324,12 @@ def test_mix_cut_snr(libri_cut):
     assert E(feats) > E(feats_snr)
 
 
+def test_mix_cut_with_other_raises_error(libri_cut):
+    libri_cut = libri_cut.drop_features()
+    with pytest.raises(ValueError):
+        _ = libri_cut.mix(libri_cut.recording)
```

Comment:
why is this test supposed to fail? because of `drop features` or because the argument to `mix` is of type `Recording`?